test/study-webidl.js

```diff
@@ -19,13 +20,32 @@ describe('The Web IDL analyser', async _ => {
     return studyWebIdl(crawlResult);
   }
 
+  function assertEqual(report, ...params) {
```

Comment:
The name is confusing since it's not really checking equality in the general sense. Maybe `assertReportMatch`?

Reply:
I switched to another approach with an `assertAnomaly` function that compares objects directly instead of the more atomic approach I had.